plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -369,7 +371,10 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
         int debugPort = getDebugPort();
         setupDebugTarget(devServerRunConfiguration, launch, debugPort, monitor);
       }
-      IVMInstall vmInstall = getVMInstall(configuration);
+      
+      IJavaProject javaProject = JavaCore.create(modules[0].getProject());
```

Comment:
I can think of a case with multiple modules, e.g., a user selects multiple projects and does _Run As_ > _App Engine_), and some project may be Java 8 while others may be Java 7. Perhaps we should iterate over `modules`?